Assets/Scripts/FloatingOrigin/Origin.cs

```diff
@@ -31,6 +33,11 @@ public class Origin : MonoBehaviour, IHasCoordinate
 
         public bool LogShifts = false;
         
+        private void Awake()
+        {
+            current = this;
```

Comment:
Can you add the Singleton boilerplate here for destroying the object if you try to make another while the singleton field property is populated? Of course we may assume no one will try this, but then again: assumptions are the mother of all fuck ups 😈

Reply:
See comment: https://github.com/Netherlands3D/twin/pull/198/files#r1691404364